modules/app-gateway-module/config.tf

```diff
@@ -0,0 +1,47 @@
+locals {
+  vnet_rg   = "aks-infra-${var.env}-rg"
+  vnet_name = "core-${var.env}-vnet"
+
+  criticality = {
+    sbox     = "Low"
+    aat      = "High"
+    prod     = "High"
+    ithc     = "Medium"
+    perftest = "Medium"
+  }
+
+  env_display_names = {
+    sbox     = "Sandbox"
+    aat      = "Staging"
+    prod     = "Production"
+    ithc     = "ITHC"
+    perftest = "Test"
+  }
+
+  tags = {
```

Comment:
Can we make this consistent with the repository? We want to pass the tags from component rather than directly embedded in the module.
Reason: For future usability of the module and If someone wants to use the module passing their own tags then they will have to change the module rather than the component.

Reply:
oops. copy paste